packages/expo-passkeys/package.json

```
@@ -19,6 +19,9 @@
   "author": "Clerk",
   "main": "build/index.js",
   "types": "build/index.d.ts",
+  "files": [
+    "build"
+  ],
```

Do we need the expo-related files at the root? 🤔 
```suggestion
  "files": [
    "build",
    "app.json",
    "expo-module.config.json"
  ],
```

---

@AlexNti Do you know which files are needed to be published as part of this module?

---

Moved this change to a separate PR: https://github.com/clerk/javascript/pull/4507